specification/powerplatform/resource-manager/Microsoft.PowerPlatform/preview/2020-10-30-preview/enterprisePolicy.json

```diff
@@ -407,13 +408,13 @@
           "description": "The identity of the EnterprisePolicy."
         },
         "properties": {
-          "$ref": "#/definitions/ConfigurationProperties",
-          "description": "The properties that define configuration.",
+          "$ref": "#/definitions/Properties",
+          "description": "The properties that define configuration for the enterprise policy",
           "x-ms-client-flatten": true
         }
       }
     },
-    "ConfigurationProperties": {
+    "Properties": {
```

Comment:
`Properties` is a generic name. Why don't you use a specific name?

Reply:
that was the name suggested by our Architect Ray